spring-integration-core/src/test/java/org/springframework/integration/gateway/GatewayProxyFactoryBeanTests.java

```diff
@@ -514,6 +525,17 @@ interface HeadersOverwriteService {
 		Message<?> echo(String s);
 
 	}
+	
+	@Gateway
+	@interface AnnotationWithAliasForGateway {
+		@AliasFor(annotation = Gateway.class, attribute = "requestChannel")
+		String requestChannelName() default "";
+	}
+	
+	interface AliasForGatewayService {
+		@AnnotationWithAliasForGateway(requestChannelName = "foo")
+		void foo();
```

Comment:
Please, make sure every single member of the class is surrounded with blank lines.

Reply:
done

Reply:
Just for the future contributions: no reason to comment `done` if you are agree.
We will see an outcome by your next commit and its changes.
Otherwise I got a commit from you and these extra messages which really a noise in my active mail box.

Thanks for understanding!

Reply:
OK

Reply:
And you still have typed this `OK` ? 😄